dali/operators/decoder/audio/audio_decoder_op.h

```diff
@@ -89,7 +88,8 @@ class AudioDecoderCpu : public Operator<CPUBackend> {
   const float quality_ = 50.0f;
   std::vector<std::string> files_names_;
   std::vector<AudioMetadata> sample_meta_;
-  std::vector<std::vector<float>> intermediate_buffers_;
+  std::vector<vector<uint8_t>> scratch_decoder_;
+  std::vector<vector<uint8_t>> scratch_resampler_;
```

Comment:
```suggestion
  std::vector<vector<float>> scratch_resampler_;
```
It's always float, so it can be a vector of actual type used - should save us some reinterpret_casts and sizeofs.